packages/playwright/src/common/fixtures.ts

```diff
@@ -23,7 +23,7 @@ import type { FixturesWithLocation } from './config';
 export type FixtureScope = 'test' | 'worker';
 type FixtureAuto = boolean | 'all-hooks-included';
 const kScopeOrder: FixtureScope[] = ['test', 'worker'];
-type FixtureOptions = { auto?: FixtureAuto, scope?: FixtureScope, option?: boolean, timeout?: number | undefined, title?: string, box?: boolean };
+type FixtureOptions = { auto?: FixtureAuto, scope?: FixtureScope, option?: boolean, timeout?: number | undefined, title?: string, box?: boolean, location?: Location };
```

Comment:
It seems like any fixtures changes are unrelated to `test.step` locations. I think we should revert this one.